src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java

```diff
@@ -133,7 +144,7 @@ public void testDestroy() throws Exception {
         // should remove all listeners, file sets, and filters
         checker.destroy();
 
-        final File tempFile = File.createTempFile("junit", null, temporaryFolder);
+        final File tempFile = createTempFile("junit", null);
```

Comment:
Please create overlorded method that takes 1 parameter. 
Too much nulls, it pollute code.